survey/tests/locale/test_locale_normalization.py

```diff
@@ -9,15 +10,27 @@ class TestLocaleNormalization(unittest.TestCase):
 
     def test_normalization(self):
         """ We test if the messages were properly created with makemessages --no-obsolete --no-wrap. """
-        makemessages_command = [
-            "python3",
-            "manage.py",
-            "makemessages",
-            "--no-obsolete",
-            "--no-wrap",
-            "--ignore",
-            "venv",
-        ]
+        if platform.system() == "Windows":
```

Comment:
I don't like the idea of having OS specific code, so I looked what was possible to launch python3 with the same command under windows and it seems possible but the default is "py -3" so let's keep and and polish this to make it works without configuration for everyone. What do you think about :

```suggestion
        if platform.system() == "Windows":
            python_3 = ["py", "-3"]
        else:
            python_3 = ["python3"]
        makemessages_command = python_3 + [
            "manage.py",
            "makemessages",
            "--no-obsolete",
            "--no-wrap",
            "--ignore",
            "venv",
        ]
```

Reply:
Windows can't execute `python3`.
e.g. I execute on Git Bash
![CantExecutePython3](https://user-images.githubusercontent.com/11808736/56552629-d6039600-65c6-11e9-8368-122f9f48fb99.jpg)

Because [`python3.exe` doesn't exist under Windows.](https://stackoverflow.com/a/40931434).
![NotExistPython3](https://user-images.githubusercontent.com/11808736/56552393-1ca4c080-65c6-11e9-8d2f-fc7d9b077454.jpg)

Only Mac and Linux can execute `python3`.

Reply:
I found [the other way](https://github.com/DjangoGirls/tutorial/issues/317).
I create `python3.exe` like the below if `python3.exe` doen't exist.

```bash
cp c:\Python37\python.exe c:\Python37\python3.exe
```

I checked that `python3` can be execute by it under PowerShell on Windows.
But I don't like this way.
It is because searching the executable file path of python and copying a new executable file from the executable file is cumbersome for Windows users.
I recommend to add a caution into `README.md` that Windows users need to copy `python3.exe` from `python.exe` if you adopt this method.

Reply:
Sorry, I was not very clear; I wanted to use `python3` with windows too, but this is not the default for Windows and I don't want to force Windows user to have to do configuration before using the software. So let's not do that :) For the record I saw [this configuration with an ini file](https://docs.python.org/3/using/windows.html#customizing-default-python-versions) that look like it could work.

Reply:
I understood.

I read [the site(this configuration with an ini file)](https://docs.python.org/3/using/windows.html#customizing-default-python-versions).
First, My recognition is that command under Command Prompt and Powershell on Windows must set the executable file path.
[The site(this configuration with an ini file)](https://docs.python.org/3/using/windows.html#customizing-default-python-versions) means that the version of python when using the `py` command depends on the ini file or environment variables.
Therefore, the ini file can't create `python3` command.

Reply:
like this screenshot
eg. set 2 to `PY_PYTHON3` on environment variable
![PyPython](https://user-images.githubusercontent.com/11808736/56562961-a021da00-65e5-11e9-9303-30fe91c99a97.jpg)